tests/test_xlsx2json.py

```diff
@@ -88,6 +88,33 @@ def test_json_conversion_succeeds_with_invalid_metadata(self):
             assert 'name' not in novel_sample
             assert 'species' not in novel_sample['characteristics']
 
+    def test_json_validation_fails_for_large_project_title_and_project_description(self) -> None:
+        with open(self.eva_schema) as eva_schema_file:
+            eva_json_schema = json.load(eva_schema_file)
+        with open(self.biosample_schema) as biosample_schema_file:
+            biosample_json_schema = json.load(biosample_schema_file)
+        resolver = jsonschema.RefResolver.from_schema(eva_json_schema)
+        resolver.store['eva-biosamples.json'] = biosample_json_schema
+
+        # test fails for title
+        json_data = self.get_expected_json()
+        json_data['project']['title'] = self.build_large_string_of_length(600)
+
+        assert len(json_data['project']['title']) == 600
+        with self.assertRaises(jsonschema.ValidationError):
+            jsonschema.validate(json_data, eva_json_schema, resolver=resolver)
+
+        # test fails for description
+        json_data = self.get_expected_json()
+        json_data['project']['description'] = self.build_large_string_of_length(6000)
+
+        assert len(json_data['project']['description']) == 6000
+        with self.assertRaises(jsonschema.ValidationError):
+            jsonschema.validate(json_data, eva_json_schema, resolver=resolver)
```

Comment:
What doe the error looks like ?
How will that be translated in the report ?

Reply:
In the metadata_validation.txt, it has the same format as the other errors like below:

```
/project/centre
	must have required property 'centre'
/project/description
	must NOT have more than 5000 characters
```

and this is how it looks in the report:
![Screenshot from 2025-04-08 14-32-52](https://github.com/user-attachments/assets/06d01030-7a65-4f4d-b156-bbe2f7c4f017)